src/styles/components/SmartBanner.scss

```diff
@@ -0,0 +1,301 @@
+.smartbanner-show {
+  margin-top: 80px;
+}
+
+.smartbanner-show .smartbanner {
+  display: block;
+}
+
+/** Default **/
+.smartbanner {
+  position: absolute;
+  left: 0;
+  top: 0;
+  display: none;
+  width: 100%;
+  height: 80px;
+  line-height: 80px;
+  font-family: 'Helvetica Neue', sans-serif;
+  background: #f4f4f4;
+  z-index: 9998;
+  -webkit-font-smoothing: antialiased;
+  overflow: hidden;
+  -webkit-text-size-adjust: none;
+}
+
+.smartbanner-container {
+  margin: 0 auto;
+  white-space: nowrap;
+}
+
+.smartbanner-close {
+  cursor: pointer;
+  display: inline-block;
+  vertical-align: middle;
+  margin: 0 5px;
+  font-family: 'ArialRoundedMTBold', Arial;
```

Comment:
IMO we should use font that we have used in quran.com. Font family should be either `source sans pro` or `Montserrat` or combination of both.